services/QuillLMS/client/app/bundles/Staff/components/comprehension/activitySessions/sessionView.tsx

```diff
@@ -20,12 +21,34 @@ const SessionView = ({ match }) => {
     queryFn: fetchActivity
   });
 
+  // cache rule data for updates
+  const { data: rulesData } = useQuery({
+    queryKey: [`rules-${activityId}`, activityId],
+    queryFn: fetchRules
+  });
+
   // cache session data for updates
   const { data: sessionData } = useQuery({
     queryKey: [`activity-${activityId}-session-${sessionId}`, sessionId],
     queryFn: fetchActivitySession
   });
 
+  if(!rulesData) {
```

Comment:
instead of doing this, you can just do `if(!rulesData && !sessionData) on line 52 to consolidate

Reply:
Good call, will update.